.github/workflows/ci_llama_3.1_405b_fp8_tp8.yml

```diff
@@ -4,13 +4,17 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+<<<<<<< ci_new_from_nod
+name: Release: Llama 3.1 405B FP8 TP8 Benchmarking Tests
+=======
 name: Llama 3.1 405B FP8 Benchmarking Tests
+>>>>>>> ci-new
```

Comment:
There are merge conflict markers in here

We should enable the `check-merge-conflict` hook: https://github.com/pre-commit/pre-commit-hooks?tab=readme-ov-file#check-merge-conflict